### PR TITLE
Add phishing banner

### PR DESCRIPTION
**Pull request by siddharthkp**
Added a file to check for phishing sandboxes. It is added from the server and checks with the phishing microservice before adding a banner

Live - https://fmnlq.codesandbox.dev/

[image](https://fmnlq.codesandbox.dev/)

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5461/commits/28229c074a1632a5044d0e6738f62fe82e8a4076 is at https://pr5461.build.csb.dev/s/new.

File: packages/app/config/webpack.common.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* eslint-disable global-require */
 const webpack = require('webpack');
 const path = require('path');
-const fs = require('fs');
```
Comment (siddharthkp) [inline packages/app/config/webpack.common.js:4]: got a lint error because this an unused import :)

File: packages/app/src/phishing-banner.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+// Test it here: https://codesandbox.io/s/hungry-colden-4zcny?file=/src/index.js
```
Comment (siddharthkp) [inline packages/app/src/phishing-banner.js:1]: this file isn't included anywhere in client code. It is included from the server.

**Comment by siddharthkp** [comment]
@DannyRuchtie Good v1?

**Review by andrasbacsai — APPROVED** [review]
For me, looks good for v1.

**Review by DannyRuchtie — APPROVED** [review]
Not 100% about the button.
But also don't have much time this week to redesign.
Let's merge for now and I will make a separate PR  later.

It's clear enough for users

**Comment by andrasbacsai** [comment]
I renamed phishing-banner to banner because it could be scary for people.

**Pull request merged**